### PR TITLE
UI screen width going over 100% fix, projects max width fix

### PR DESCRIPTION
**Pull request by irubido**
Navbar container + logo, were creating empty space on medium screen widths - fixed

Screenshot 2020-10-26 at 15 17 48
Projects section has max width

### CODE REVIEW

**Comment by morrigan** [comment]
I will export new photos for this case

File: site/layouts/partials/header.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <header>
     <a href="/">
-        <img src="images/NodeFactory-transparent.png" alt="NodeFactory logo" class="hidden-xs hidden-sm" />
+        <img src="images/NodeFactory-transparent.png" alt="NodeFactory logo" />
     </a>
 
-    <nav class="col-md-12" id="menu-bar">
+    <nav class="col-md-12" id="menu-bar" style="max-width: 800px">
```
Comment (morrigan) [inline site/layouts/partials/header.html]: don't use inline styles

**Comment by irubido** [comment]
Screenshot 2020-10-27 at 12 43 04
Screenshot 2020-10-27 at 12 43 23
Screenshot 2020-10-27 at 12 43 45
Screenshot 2020-10-27 at 12 44 00

**Comment by irubido** [comment]
Screenshot 2020-10-27 at 13 15 58

**Review by morrigan — APPROVED** [review]
Ok, looks nice!

**Pull request merged**